optin/reporting/utils/Validation.js

```diff
@@ -0,0 +1,664 @@
+	'Europe/Istanbul',
+	'Europe/Jersey',
+	'Europe/Kaliningrad',
+	'Europe/Kiev',
+	'Europe/Kirov',
+	'Europe/Lisbon',
+	'Europe/Ljubljana',
+	'Europe/London',
+	'Europe/Luxembourg',
+	'Europe/Madrid',
+	'Europe/Malta',
+	'Europe/Mariehamn',
+	'Europe/Minsk',
+	'Europe/Monaco',
+	'Europe/Moscow',
+	'Europe/Nicosia',
+	'Europe/Oslo',
+	'Europe/Paris',
+	'Europe/Podgorica',
+	'Europe/Prague',
+	'Europe/Riga',
+	'Europe/Rome',
+	'Europe/Samara',
+	'Europe/San_Marino',
+	'Europe/Sarajevo',
+	'Europe/Saratov',
+	'Europe/Simferopol',
+	'Europe/Skopje',
+	'Europe/Sofia',
+	'Europe/Stockholm',
+	'Europe/Tallinn',
+	'Europe/Tirane',
+	'Europe/Tiraspol',
+	'Europe/Ulyanovsk',
+	'Europe/Uzhgorod',
+	'Europe/Vaduz',
+	'Europe/Vatican',
+	'Europe/Vienna',
+	'Europe/Vilnius',
+	'Europe/Volgograd',
+	'Europe/Warsaw',
+	'Europe/Zagreb',
+	'Europe/Zaporozhye',
+	'Europe/Zurich',
+	'GB',
+	'GB-Eire',
+	'GMT',
+	'GMT+0',
+	'GMT-0',
+	'GMT0',
+	'Greenwich',
+	'HST',
+	'Hongkong',
+	'Iceland',
+	'Indian/Antananarivo',
+	'Indian/Chagos',
+	'Indian/Christmas',
+	'Indian/Cocos',
+	'Indian/Comoro',
+	'Indian/Kerguelen',
+	'Indian/Mahe',
+	'Indian/Maldives',
+	'Indian/Mauritius',
+	'Indian/Mayotte',
+	'Indian/Reunion',
+	'Iran',
+	'Israel',
+	'Jamaica',
+	'Japan',
+	'Kwajalein',
+	'Libya',
+	'MET',
+	'MST',
+	'MST7MDT',
+	'Mexico/BajaNorte',
+	'Mexico/BajaSur',
+	'Mexico/General',
+	'NZ',
+	'NZ-CHAT',
+	'Navajo',
+	'PRC',
+	'PST8PDT',
+	'Pacific/Apia',
+	'Pacific/Auckland',
+	'Pacific/Bougainville',
+	'Pacific/Chatham',
+	'Pacific/Chuuk',
+	'Pacific/Easter',
+	'Pacific/Efate',
+	'Pacific/Enderbury',
+	'Pacific/Fakaofo',
+	'Pacific/Fiji',
+	'Pacific/Funafuti',
+	'Pacific/Galapagos',
+	'Pacific/Gambier',
+	'Pacific/Guadalcanal',
+	'Pacific/Guam',
+	'Pacific/Honolulu',
+	'Pacific/Johnston',
+	'Pacific/Kiritimati',
+	'Pacific/Kosrae',
+	'Pacific/Kwajalein',
+	'Pacific/Majuro',
+	'Pacific/Marquesas',
+	'Pacific/Midway',
+	'Pacific/Nauru',
+	'Pacific/Niue',
+	'Pacific/Norfolk',
+	'Pacific/Noumea',
+	'Pacific/Pago_Pago',
+	'Pacific/Palau',
+	'Pacific/Pitcairn',
+	'Pacific/Pohnpei',
+	'Pacific/Ponape',
+	'Pacific/Port_Moresby',
+	'Pacific/Rarotonga',
+	'Pacific/Saipan',
+	'Pacific/Samoa',
+	'Pacific/Tahiti',
+	'Pacific/Tarawa',
+	'Pacific/Tongatapu',
+	'Pacific/Truk',
+	'Pacific/Wake',
+	'Pacific/Wallis',
+	'Pacific/Yap',
+	'Poland',
+	'Portugal',
+	'ROC',
+	'ROK',
+	'Singapore',
+	'Turkey',
+	'UCT',
+	'US/Alaska',
+	'US/Aleutian',
+	'US/Arizona',
+	'US/Central',
+	'US/East-Indiana',
+	'US/Eastern',
+	'US/Hawaii',
+	'US/Indiana-Starke',
+	'US/Michigan',
+	'US/Mountain',
+	'US/Pacific',
+	'US/Pacific-New',
+	'US/Samoa',
+	'UTC',
+	'Universal',
+	'W-SU',
+	'WET',
+	'Zulu'
+];
+
+const Validation = {
+	validate: method => {
+		const paginationValidation = [
+			query('pageSize', 'pageSize required').exists().isInt(),
+			query('pageNumber', 'pageNumber required').exists().isInt()
+		];
+
+		const sortValidation = [
+			query('sortBy', 'field name must be string')
```

Comment:
should you limit possible sortBy values?

Reply:
Agree, i will add that 👍🏼